sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -65,9 +65,12 @@ def _get_blob_name(blob):
     :rtype: str
     """
     try:
-        return blob.get('name')
```

Comment:
I still feel get('name') makes more sense here.... a customer complained about using blob.name
https://github.com/Azure/azure-sdk-for-python/issues/10886

Reply:
Yeah - On further assessment I think this is okay - we can keep `blob.get('name')` - it seems to be a pattern we are already using elsewhere - so if it was going to break it already would have done so :)